src/AppBundle/Form/LoginType.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace AppBundle\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
+use Symfony\Component\Form\Extension\Core\Type\PasswordType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class LoginType extends AbstractType
```

Comment:
@HeahDude a simple html form is enough for login. we don't need to maintain anything else (eq a class for a formType)